cmd/loadit/main.go

```diff
@@ -27,6 +27,7 @@ var (
 	flagKeepaliveInterval = flag.Int("keepalive-interval", agent.DefaultKeepaliveInterval, "Keepalive interval")
 	flagKeepaliveTimeout  = flag.Int("keepalive-timeout", types.DefaultKeepaliveTimeout, "Keepalive timeout")
 	flagProfilingPort     = flag.Int("pprof-port", 6060, "pprof port to bind to")
+	flagPromPort          = flag.String("prom", ":8080", "binding for prometheus server")
```

Comment:
s/binding/port?

Reply:
Nah, binding, it's host+port

Reply:
Perhaps we should rename the flag to `flagPromBinding` to accurately reflect what the flag is for?